dts/bindings/led_strip/worldsemi,ws2812-rpi_pico-pio.yaml

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) 2023, TOKITA Hiroshi
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  The pio node configured for ws2812.
+
+compatible: "worldsemi,ws2812-rpi_pico-pio"
+
+include: pinctrl-device.yaml
+
+properties:
+  bit-waveform:
+    type: array
+    description: |
+      This property defines the waveform for sending 1-bit data.
+      The program uses the first three elements of the array.
+      Where, let these value be T0, T1, and T2.
```

Comment:
values ^
And I am not sure about the "Where, " part. You should probably remove it.

Reply:
Removed the "Where, " part.